distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -0,0 +1,70 @@
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<configuration xmlns="urn:activemq"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="urn:activemq /schema/artemis-configuration.xsd">
+
+    <jms xmlns="urn:activemq:jms">
+        <queue name="DLQ"/>
+        <queue name="ExpiryQueue"/>
+    </jms>
+
+    <core xmlns="urn:activemq:core">
+
+        <!-- this could be ASYNCIO or NIO -->
+        <journal-type>NIO</journal-type>
+        <paging-directory>./data/paging</paging-directory>
+        <bindings-directory>./data/bindings</bindings-directory>
+        <journal-directory>./data/journal</journal-directory>
+        <large-messages-directory>./data/large-messages</large-messages-directory>
+        <journal-min-files>10</journal-min-files>
+
+        <!--
+         This value was determined through a calculation.
+         Your system could perform 0.63 writes per millisecond
+         on the current journal configuration.
+         That translates as a sync write every 1591999 nanoseconds
+        -->
+        <journal-buffer-timeout>1591999</journal-buffer-timeout>
+
+        <acceptors>
+            <!-- AMQP Acceptor.  Listens on default AMQP port for AMQP traffic.-->
+            <acceptor name="amqp">tcp://0.0.0.0:5672?protocols=AMQP</acceptor>
+        </acceptors>
+
+        <security-settings>
+            <security-setting match="#">
+                <permission type="createNonDurableQueue" roles="manager"/>
+                <permission type="deleteNonDurableQueue" roles="manager"/>
+                <permission type="createDurableQueue" roles="manager"/>
+                <permission type="deleteDurableQueue" roles="manager"/>
+                <permission type="consume" roles="manager"/>
+                <permission type="send" roles="manager"/>
+                <!-- we need this otherwise ./artemis data imp wouldn't work -->
+                <permission type="manage" roles="manager"/>
+            </security-setting>
+        </security-settings>
+
+        <address-settings>
+            <!--default for catch all-->
+            <address-setting match="#">
+                <dead-letter-address>jms.queue.DLQ</dead-letter-address>
+                <expiry-address>jms.queue.ExpiryQueue</expiry-address>
+                <redelivery-delay>0</redelivery-delay>
+                <max-size-bytes>10485760</max-size-bytes>
```

Comment:
How was this value arrive at? What happens if a larger message is sent? Can it be configured?